plugin/src/utils/remix_storage.ts

```diff
@@ -19,3 +19,25 @@ export const getAllContractFiles = async (remixClient: RemixClient, path: string
   }
   return files
 }
+
+export const getContractTargetPath = (allContracts: ContractFile[], contractFilePath: string) => {
+  for (const { file_name } of allContracts) {
+    if (file_name.includes(contractFilePath)) {
+      const parts = file_name.split('/')
+      if (parts.length === 1) {
+        return './'
+      } else {
+        parts.pop()
+        return './' + parts.join('/')
+      }
+    }
+  }
+
+  return './'
+}
+
+export const findFilesNotInContracts = (allContracts: ContractFile[]) => {
```

Comment:
@satyambnsal do we want to find all the files or only contracts?

Reply:
Only contract files.